notification-service/src/main/java/com/scaler/notificationservice/controller/NotificationController.java

```diff
@@ -0,0 +1,35 @@
+package com.scaler.notificationservice.controller;
+
+import com.scaler.notificationservice.model.Notification;
+import com.scaler.notificationservice.service.EmailService;
+import com.scaler.notificationservice.service.SMSService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("v1/api/notifications")
+public class NotificationController implements NotificationOperation {
+
+    @Autowired
+    private EmailService emailService;
+
+    @Autowired
+    private SMSService smsService;
+
+
+    @Override
```

Comment:
do you really need an api for sending notification? sending notification will be triggered by an incoming amq message

Reply:
Just for Testing